GeoLib/Station.cpp

```diff
@@ -23,12 +24,12 @@
 namespace GeoLib
 {
 Station::Station(double x, double y, double z, std::string name) :
```

Comment:
It was not before... but see the answer for the question on the next line (in the diff).